src/platform/android/AndroidChipPlatform-JNI.cpp

```diff
@@ -60,6 +60,9 @@ static bool JavaBytesToUUID(JNIEnv * env, jbyteArray value, chip::Ble::ChipBleUU
 namespace {
 JavaVM * sJVM = nullptr;
 JniGlobalReference sAndroidChipPlatformExceptionCls;
+jclass callbackClass;
+jmethodID onLogMessageMethod;
```

Comment:
Add `s` prefix for module globals (statics)

Reply:
I will fix it.